server/src/infra/entities/person.entity.ts

```diff
@@ -30,6 +30,9 @@ export class PersonEntity {
   @Column({ default: '' })
   name!: string;
 
+  @Column({ type: 'date', nullable: true })
+  birthDate?: Date | null;
```

Comment:
```suggestion
  birthDate!: Date | null;
```

This is not optional, it will/should always be present.